labgrid/driver/shelldriver.py

```diff
@@ -111,6 +115,13 @@ def _await_login(self):
 
         start = time.time()
 
+        def remaining_time():
```

Comment:
This can also be simplified by using the Timeout helper (although here we already incorrectly use `time.time()`).